pkg/tuned/host_payload.go

```diff
@@ -79,7 +79,7 @@ Environment=FG=--foreground
 # ex: PF=--pidfile /run/stalld.pid
 Environment=PF="--pidfile /run/stalld.pid"
 
-ExecStart=/usr/local/bin/stalld $CLIST $AGGR $BP $BR $BD $THRESH $LOGGING $FG $PF
+ExecStart=chrt -f 10 /usr/local/bin/stalld $CLIST $AGGR $BP $BR $BD $THRESH $LOGGING $FG $PF
```

Comment:
nit: `chrt` or `/usr/bin/chrt`?

> ExecStart=
> > For each of the specified commands, the first argument must be either an absolute path to an executable or a simple file name without any slashes.

https://www.freedesktop.org/software/systemd/man/systemd.service.html

maybe it used not to be allowed to rely on `$PATH`, or maybe I have something wrong in mind!
but looking at `/usr/lib/systemd/system`, the absolute path seems to be the common usage ... except for systemd itself!

Reply:
Yeah, good point. I was thinking about this when I did the PR.  I decided not to use the absolute path just in case the binary is moved somewhere else to allow for more flexibility.  However, chances it moves are actually pretty slim.  If you think absolute path is better, I don't have a problem with this.